composer.json

```diff
@@ -65,17 +18,20 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "yiisoft/arrays": "^3.0@dev",
-        "yiisoft/strings": "^3.0@dev",
+        "php": "^7.4",
         "ext-pdo": "*",
-        "php": "^7.4"
+        "yiisoft/arrays": "^3.0@dev",
+        "yiisoft/strings": "^3.0@dev"
     },
     "require-dev": {
         "yiisoft/yii-console": "^3.0@dev",
         "yiisoft/active-record": "^3.0@dev",
+        "yiisoft/aliases": "^3.0@dev",
         "yiisoft/db-mysql": "^3.0@dev",
         "yiisoft/db-sqlite": "^3.0@dev",
+        "yiisoft/db-pgsql": "^3.0@dev",
         "yiisoft/cache": "^3.0@dev",
```

Comment:
Cache seems to be used in Schema, Query and Command so should be moved to `require`. `cache-file` is fine in `require-dev`.